web3/queries/votes/getDecodedAdminTransactions.ts

```diff
@@ -0,0 +1,32 @@
+import {
+  DecodedAdminTransactionsByIdentifierT,
+  DecodedAdminTransactionsT,
+} from "types";
+
+export async function getDecodedAdminTransactions(
+  decodedIdentifiers: string[]
+) {
+  const response = await fetch("/api/decode-admin", {
+    method: "POST",
+    headers: {
+      "Content-Type": "application/json",
+    },
+    body: JSON.stringify({ identifiers: decodedIdentifiers }),
+  });
+
+  if (!response.ok) {
+    throw new Error("Error fetching decoded admin transactions");
+  }
+
+  const result = (await response.json()) as DecodedAdminTransactionsT[];
```

Comment:
if we have a shape for this data we could use a validator to enforce this, probably a todo item